addon/components/sortable-group.js

```diff
@@ -13,6 +13,18 @@ export default Component.extend({
 
   attributeBindings: ['data-test-selector'],
 
+  /**
+    Called when order of items has been changed
+
+    @property onChange
+    @type Action
+    @param object group model (omitted if not set)
+    @param array item models in their new order
+    @param object item model just dragged
+    @default null
+  */
+  onChange: ()=>{},
```

Comment:
`onChange: () => { },`

Reply:
OK, maybe we can have eslint rule for that?

Reply:
And incorporate Prettier into the project :)